src/index.js

```diff
@@ -39,22 +38,6 @@ export default ({
 
     return new Promise((resolve, reject) => {
       const promiseCreators = action[CHAIN_API].map((createCallApiAction) => {
-        const apiAction = createCallApiAction()[CALL_API]
```

Comment:
The key change is don't call `createCallApiAction` here, the actions should only be called on https://github.com/CodementorIO/redux-api-middleman/blob/598d554f5691e96f4bebd3d7b3b76bafbec1126e/src/index.js#L54